lib/scout_apm/instruments/action_controller_rails_2.rb

```diff
@@ -38,7 +39,8 @@ def install
     module ActionControllerRails2Instruments
       def self.included(instrumented_class)
         # XXX: Don't lookup context by global
-        ScoutApm::Agent.instance.context.logger.info "Instrumenting #{instrumented_class.inspect}"
+        @agent_context = ScoutApm::Agent.instance.context
+        @agent_context.logger.info "Instrumenting #{instrumented_class.inspect}"
```

Comment:
Don't create an instance variable here - we're inside the context of customer's controllers at this point, and we need to be careful not to pollute their namespace with extra instance variables.

Looking up by the global is fine and doesn't need to be changed.